packages/node_modules/samples/browser-plugin-meetings/index.html

```diff
@@ -3,438 +3,399 @@
+                <button id="meetings-add-media" type="button" onclick="addMedia()" class="btn-code">meeting.addMedia()</button>
+              </div>
+          </fieldset>
+          <!-- meetings / current -->
+          <form id="current-meeting">
+            <fieldset>
+                <legend>Current Meeting</legend>
+                <p id="meetings-current-details" class="note">NOTE: Not in a meeting</p>
+                <div class="u-mv">
+                  <button id="meetings-add-media" type="button" onclick="addMedia()" class="btn-code">meeting.addMedia()</button>
+                  <button id="meetings-leave" type="button" onclick="leaveMeeting()" class="btn-code">meeting.leave()</button>
+                </div>
+            </fieldset>
+          </form>
+          <!-- meetings / incoming -->
+          <fieldset id="incomingsection" style="visibility: hidden;">
+            <legend>Incoming Call</legend>
+            <div class="u-mv">
+                <button onclick="answerMeeting(this)" id="answer" class="btn--green">Answer</button>
+                <button onclick="rejectMeeting(this)" id="reject" class="btn--red">Reject</button>
             </div>
-            <textarea id="gc-transcription-content" disabled>Transcription Content: Webex Assistant must be enabled, check the console!</textarea>
-          </div>
-        </fieldset>
-      </form>
-
-      <br />
-
-      <form id="source-devices-controls">
-        <fieldset>
-            <legend>Get Media Devices</legend>
-            <div>
-              <button id="sd-get-media-streams" type="button" onclick="getMediaStreams()">meeting.getMediaStreams()</button> <label>Get media streams with default devices</label>
-              <br><br>
-              <button id="sd-get-media-devices" type="button" onclick="clearMediaDeviceList(); getMediaDevices()">meeting.getDevices()</button>
-              <p>
-                The first time the browser loads the webpage, media access permissions have not been set.
-                Therefore, using a valid <code>meeting</code> object, invoke the <code>getMediaStreams</code> function.
-                This will ask the user for permission to their microphone and camera. Once permission has been granted, invoke the <code>getDevices</code> function to get a list of media devices.
-              </p>
-              <p>
-                  After the browser gets the user's permission to access their media devices, on subsequent web page loads any  calls to <code>getDevices</code> will work as expected.
-              </p>
+          </fieldset>
+        </section>
+    
+        <section>
+          <h2>Meeting Controls</h2>
+          <p>Once in a meeting utilize these controls to manipulate the meeting.</p>
+          <p>Please note that you must be a meeting moderator in order to lock or record a meeting.</p>
+          <form id="general-controls">
+            <fieldset>
+              <legend>General Controls</legend>
```

Comment:
```suggestion
              <legend>Lock/Unlock</legend>
```

Maybe something to address on the meeting controls PR